File: web3rpc/rpc-specs/code-samples/javascript/klay/account/getAccount.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const address = '0x1cbd3b2770909d4e10f157cabc84c7264073c9ec'
    const blockNumberOrHash = 'latest'
    
    sdk.klay.getAccount(address, blockNumberOrHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/kaia/account/getAccount.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x3111a0577f322e8fb54f78d9982a26ae7ca0f722"
blockNumberOrHash = "latest"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_account(address, blockNumberOrHash)

print(kaia_response)
